routes/story.route.server.js

```diff
@@ -384,16 +384,20 @@ module.exports = app => {
         }
     };
 
+    let getSortedByFlagged = (req, res) => storyDao.getSortedFlagged(req.params.numberOfStories).then(stories => res.json(stories));
+
```

Comment:
I think we should have a try catch here because the parseInt in the story.dao.server.js can throw an error

Reply:
followed the one similar to findTopStories